test/lib/ansible_test/_util/controller/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -84,6 +84,22 @@ def date(error_code=None):
     return Any(isodate, error_code=error_code)
 
 
+def require_only_one(keys):
+    def f(obj):
+        found = None
+        for k in obj.keys():
+            if k in keys:
+                if k is None:
```

Comment:
Shouldn't this line be `if found is None`? I think right now the error message "Found conflicting keys" is raised **every time any one of the keys is used**.

Reply:
Created #83511 to fix that.